### DISCUSSION TITLE
Considerations, limitations for large file collections

### DISCUSSION THREAD

**Discussion by ScottCov** [dadoonet/fscrawler#1597] [General]
I have a collection of around 200,000 pdfs located in about 300 subfolders and total size of the files is a little more than a terabyte. I am currently using docker which is a bit new to me but from what I read read even if I bind or create volumes all the pdfs would still need to be present inside the container for this to work in which case my page file will need to grow to accomodate that size? If so, could I avoid this by using the non-dockerized version?

**Comment by dadoonet** [comment]
Basically, when you "mount" a volume in Docker, you don't have to allocate space for this. Docker will just use the mounted volume without copying anything.

**Reply by ScottCov** [reply]
OK thank you David I subsequently discovered I may have another issue causing the memory problem outside of docker.

Scott
